lib/rmt/cli/systems.rb

```diff
@@ -49,4 +52,70 @@ def remove(target)
     raise RMT::CLI::Error.new(_('System with login %{login} not found.') % { login: target })
   end
   map 'rm' => :remove
+
+  desc 'purge', _('Removes inactive systems')
+  option :before, aliases: '-b', type: :string, desc: _('Remove systems before the given date (format: "<year>-<month>-<day>")')
+  option :non_interactive, aliases: '-n', type: :boolean, default: false, desc: _("Don't ask for anything")
+  long_desc <<~PURGE
+    #{_('Removes old systems and their activations if they are inactive.')}
+
+    #{_('By default, inactive systems are those that have not contacted in any way with RMT for the past two months. You can override this with the \'-b / --before\' flag.')}
+
+    #{_('The command will list you the candidates for removal and will ask for confirmation. You can tell this subcommand to go ahead without asking with the \'-n / --non-interactive\' flag.')}
+
+    #{_('Examples')}:
+
+    $ rmt-cli systems purge --non-interactive --before 2022-02-28
+  PURGE
+  def purge
+    ask, before = purge_options
+    systems = System.where('last_seen_at < ?', before)
+
+    decorator = RMT::CLI::Decorators::SystemDecorator.new(systems)
+
```

Comment:
Could we add an output here, like "Purging systems that have not contacted this RMT since ..."?